### PR TITLE
blog(@embark/quorum): Using Embark with Quorum

### PR DESCRIPTION
**Pull request by emizzle**

### CODE REVIEW

**Review by 0x-r4bbit — APPROVED** [review]
Really nice! 👍

File: source/_posts/2020-03-14-using-embark-with-quorum.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+  "embark-quorum": "5.3.0"
+}
+```
+Then install the package with your package manager.
+```
+# /bin/bash
+npm i
+// OR
+yarn
+```
```
Comment (jrainville) [inline source/_posts/2020-03-14-using-embark-with-quorum.md]: This whole section can be replaced by: 
You can run:
```
yarn add embark-quorum
// OR
npm install embark-quorum --save
```
(add the @5.3.0) if you want people to be sure of the version.

File: source/_posts/2020-03-14-using-embark-with-quorum.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
+title: Enterprise &ETH;Apps with Quorum
+summary: "Quorum is an open source blockchain platform that combines the innovation of the public Ethereum community with enhancements to support enterprise needs. Learn how to use Embark to deploy contracts privately and make private transactions in a Quorum network, facilitating enterprise-level &ETH;Apps."
+author: eric_mastro
+categories:
```
Comment (jrainville) [inline source/_posts/2020-03-14-using-embark-with-quorum.md]: The first category would be tutorial IMO

**Review by jrainville — COMMENTED** [review]
Nice job. Since this is a copy of the readme, should we change the readme to point to this tutorial instead, so we don't have to update it twice?
On the readme, we could keep only the installation guide with pre-requisites.

**Pull request closed without merging**